sympy/functions/elementary/tests/test_piecewise.py

```diff
@@ -304,7 +304,40 @@ def test_piecewise_solve():
                   (-x + 2, x - 2 <= 0), (x - 2, x - 2 > 0))
     assert solve(g, x) == [5]
 
-# See issue 4352 (enhance the solver to handle inequalities).
+
+def test_piecewise_solveset():
```

Comment:
This should be in the solveset test suite; you aren't testing Piecewise, you are testing how solveset handles it.